### PR TITLE
feat: add acm/idm

### PR DESCRIPTION
**Pull request by lvermeulen**

### CODE REVIEW

**Review by emalfroy — DISMISSED** [review]

File: src/StreetNameRegistry.Migrator.StreetName/paket.references [hunk 1]
```diff
@@ -21,6 +21,7 @@ Serilog.Enrichers.Thread
 Serilog.Enrichers.Environment
 Serilog.Formatting.Compact
 
+Microsoft.AspNetCore.Authentication.JwtBearer
```
Comment (ArneD) [inline src/StreetNameRegistry.Migrator.StreetName/paket.references]: Does the migrator really need this?

File: paket.dependencies [hunk 2]
```diff
@@ -26,12 +26,15 @@ nuget Dapper 2.0.123
 nuget AsyncEnumerator 4.0.2
 nuget Destructurama.JsonNet 2.0.0
 
+nuget Microsoft.AspNetCore.Authentication.JwtBearer 6.0.11
```
Comment (pgallik) [inline paket.dependencies]: Isn't this a sub dependency of Basisregisters.AcmIdm and therefore indirectly included?

**Review by pgallik — DISMISSED** [review]

File: src/StreetNameRegistry.Migrator.StreetName/paket.references [hunk 3]
```diff
@@ -1,4 +1,4 @@
-
+Be.Vlaanderen.Basisregisters.AcmIdm
```
Comment (ArneD) [inline src/StreetNameRegistry.Migrator.StreetName/paket.references]: same here

**Review by pgallik — DISMISSED** [review]

File: src/StreetNameRegistry.Api.BackOffice/StreetNameController-CorrectApproval.cs [hunk 4]
```diff
@@ -41,6 +44,7 @@ public partial class StreetNameController
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
         [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = PolicyNames.Adres.InterneBijwerker)]
```
Comment (jvandaal) [inline src/StreetNameRegistry.Api.BackOffice/StreetNameController-CorrectApproval.cs:48]: I think we need to register both policies (Adres.DecentraleBijwerker en Adres.InterneBijwerker) as separate attributes.

File: src/StreetNameRegistry.Api.BackOffice/StreetNameController-CorrectRejection.cs [hunk 5]
```diff
@@ -41,6 +44,7 @@ public partial class StreetNameController
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
         [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = PolicyNames.Adres.InterneBijwerker)]
```
Comment (jvandaal) [inline src/StreetNameRegistry.Api.BackOffice/StreetNameController-CorrectRejection.cs:48]: Same comment as previous (two policies).

File: src/StreetNameRegistry.Api.BackOffice/StreetNameController-CorrectRetirement.cs [hunk 6]
```diff
@@ -42,6 +45,7 @@ public partial class StreetNameController
         [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
         [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
         [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamples))]
+        [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme, Policy = PolicyNames.Adres.InterneBijwerker)]
```
Comment (jvandaal) [inline src/StreetNameRegistry.Api.BackOffice/StreetNameController-CorrectRetirement.cs:49]: Same comment as previous (two policies).

File: test/StreetNameRegistry.Tests/paket.references [hunk 7]
```diff
@@ -6,6 +6,8 @@ xunit.categories
 AutoFixture
 FluentAssertions
 
+Microsoft.AspNetCore.Authentication.JwtBearer
```
Comment (jvandaal) [inline test/StreetNameRegistry.Tests/paket.references]: Is this necessary?

Comment (lvermeulen) [reply test/StreetNameRegistry.Tests/paket.references]: removed, pr is building again

**Review by jvandaal — APPROVED** [review]

**Pull request merged**